### ISSUE TITLE
Blazor Individual Accounts Page Logout link doesn't work

### ISSUE THREAD

**Issue by Tratcher** [aspnet/AspNetCore#16721]
### Describe the bug
The logout link on https://localhost:44318/Identity/Account/Manage doesn't work, 
This looks like a regression caused by https://github.com/aspnet/AspNetCore/pull/12419/files, which changed some of the logout links but not others.

_Login_artial.cshtml:
`<a class="nav-link text-dark" asp-area="Identity" asp-page="/Account/LogOut">Logout</a>`

LoginDisplay.razor:
```
        <form method="post" action="Identity/Account/LogOut">
            <button type="submit" class="nav-link btn btn-link">Log out</button>
        </form>
```

### To Reproduce
1. Using this version of ASP.NET Core 3.0.0
2. Create a new Blazor template with Individual Accounts.
3. Register/Login
4. Go to the account management page
5. Log out

Result: It returns a 200 without signing out and you get a blank page with only the header.

### Expected behavior
You should be logged out and redirected to the home page.

@blowdart @SteveSandersonMS

**Comment by javiercn** [comment]
@Tratcher i believe this has been fixed and it's already available on the latest nightly. (I tried to repro it just now and I couldn't).

What version are you using?

**Comment by javiercn** [comment]
```razor
@using Microsoft.AspNetCore.Identity
@inject SignInManager<IdentityUser> SignInManager
@inject UserManager<IdentityUser> UserManager
@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers

<ul class="navbar-nav">
@if (SignInManager.IsSignedIn(User))
{
    <li class="nav-item">
        <a  class="nav-link text-dark" asp-area="Identity" asp-page="/Account/Manage/Index" title="Manage">Hello @User.Identity.Name!</a>
    </li>
    <li class="nav-item">
        <form class="form-inline" asp-area="Identity" asp-page="/Account/Logout" asp-route-returnUrl="/" method="post">
            <button  type="submit" class="nav-link btn btn-link text-dark">Logout</button>
        </form>
    </li>
}
else
{
    <li class="nav-item">
        <a class="nav-link text-dark" asp-area="Identity" asp-page="/Account/Register">Register</a>
    </li>
    <li class="nav-item">
        <a class="nav-link text-dark" asp-area="Identity" asp-page="/Account/Login">Login</a>
    </li>
}
</ul>
```

The login partial as scaffolded by the template

**Comment by Tratcher** [comment]
Yes I was using 3.0.0. Looks like it was fixed by #13679.
